src/celeritas/phys/PhysicsStepUtils.hh

```diff
@@ -77,12 +77,14 @@ calc_physics_step_limit(MaterialTrackView const& material,
 
     // Determine limits from discrete interactions
     StepLimit limit;
-    limit.step = 0;
     limit.action = physics.scalars().discrete_action();
-    if (!particle.is_stopped())
+    if (particle.is_stopped())
+    {
+        limit.step = 0;
+    }
+    else
     {
         limit.step = physics.interaction_mfp() / total_macro_xs;
```

Comment:
If `num_particle_processes() == 0`, then `total_macro_xs == 0`, so we're dividing by zero. My lldb shows `limit.step: +Inf`---why is this not crashing?

Reply:
Dividing a nonzero number by zero is a well-defined operation for IEEE floating point arithmetic (but not for integers, which *does* crash as you say, depending on low-level processor flags). So it's consistent and correct that a zero cross section results in an infinite step.

Reply:
Good to know, I thought any division by zero resulted in `NaN` or undefined (not just `0/0`).